relation_engine/batchload/load_utils.py

```diff
@@ -92,16 +90,16 @@ def process_edge(edge, node_collection, load_version, timestamp):
     """
     edge.update({
             '_key':             edge['id'] + '_' + load_version,
-            '_from':            node_collection + '/' + edge['from'] + '_' + load_version,
-            '_to':              node_collection + '/' + edge['to'] + '_' + load_version,
+            '_from':            edge['from'] + '_' + load_version,
```

Comment:
I thought "from" and "to" do not include collection name? Not sure how you can set "_from" and "_to" without that. Is it an arangodb 3.5 thing?

Reply:
You use the `--from-collection-prefix` (and `-to-`...) `arangoimport` arguments to specify the node collection.